ui/src/app/utils/navigation.tsx

```diff
@@ -61,8 +60,24 @@ export const useNavigation = () => {
 
 export const startTitleSetter = () => {
   document.title = buildPageTitleForEnvironment();
-  routeDataStore.subscribe(({title, pathElementForTitle}) => {
-    document.title = buildPageTitleForEnvironment(title || urlParamsStore.getValue()[pathElementForTitle]);
+  routeDataStore.subscribe(({title}) => {
+    // Currently, the only path element we use in the title is notebook name.
+    // We'll need to revert to using pathElementForTitle in route data and
+    // put more matchPath if this ever changes.
+    let nbName;
```

Comment:
No, I lied, this is the most egregious `matchPath`. Getting rid of pathElementForTitle might actually be controversial.

We could alternatively do this in `withRouteData` since that has access to match params - thoughts?

Reply:
Doesn't look like they do.